README.md

```diff
@@ -214,6 +214,7 @@ voices are heard.
 > * Be respectful.
 > * Be mindful of the words you choose.
 > * When we disagree, try to understand why.
+> * Assume positive intent
```

Comment:
Sorry to nitpick, but the other points all end in a period. (Yeah, sorry, I'm that guy)